ibis/expr/operations/arrays.py

```diff
@@ -186,3 +186,18 @@ def dtype(self):
                 }
             )
         )
+
+
+class Range(Value):
```

Comment:
I assume this base class is for the follow-up range implementations.

Reply:
Yep!